### DISCUSSION TITLE
Notes Button on Left Side

### DISCUSSION THREAD

**Discussion by rodneyljr** [ezra-bible-app/ezra-bible-app#1157] [Ideas]
It would be great to have a notes section that isn't tied to a specific verse. A place where I can keep my prayer list, verses I like to call upon. That kind of thing.

**Comment by tobias-klein** [comment]
Hi @rodneyljr 
That would be like what I proposed in #137. However, that's a major feature and a lot of work to implement. Not sure I can do that anytime soon ...

**Comment by rodneyljr** [comment]
Totally understand. Thanks for letting me know!
